user.go

```diff
@@ -328,6 +393,43 @@ func (c *Client) CurrentUsersPlaylistsOpt(opt *Options) (*SimplePlaylistPage, er
 	return &result, nil
 }
 
+// CurrentUsersPlaylistsAllOpt is a convenience method for CurrentUsersPlaylistsOpts that
+// returns a SimplePlaylist slice from all SimplePlaylistPages with options
+func (c *Client) CurrentUsersPlaylistsAllOpt(opt *Options) ([]SimplePlaylist, error) {
+	r, err := c.CurrentUsersPlaylistsOpt(opt)
+	if err != nil {
+		return nil, err
+	}
+
+	playlists := make([]SimplePlaylist, len(r.Playlists))
+	copy(playlists, r.Playlists)
+
+	for {
+		err = r.NextPage(c)
+		if err == ErrNoMorePages {
+			break
+		} else if err != nil {
+			return nil, err
+		}
+		playlists = append(playlists, r.Playlists...)
+	}
+
+	return playlists, nil
+}
+
+// CurrentUsersTopArtists is like CurrentUsersTopArtistsOpt but with
+// sensible defaults. The default limit is 20 and the default timerange
+// is medium_term.
+func (c *Client) CurrentUsersTopArtists() (*FullArtistPage, error) {
+	return c.CurrentUsersTopArtistsOpt(nil)
+}
+
+// CurrentUsersTopArtistsAll is a convenience methode that
```

Comment:
s/methode/method

Reply:
@zmb3, hmm?

Reply:
Sorry, that's my way of pointing out a typo :-)

`methode` is spelled incorrectly.  You want to replace it with `method`.